doc/fileformat/nmodl.rst

```diff
@@ -334,18 +334,18 @@ Nernst
 Many mechanisms make use of the reversal potential of an ion (``eX`` for ion ``X``).
 A popular equation for determining the reversal potential during the simulation is
 the `Nernst equation <https://en.wikipedia.org/wiki/Nernst_equation>`_.
-Both Arbor and NEURON make use of ``nernst``. Arbor implements it as a mechanism and
+Both Arbor and NEURON make use of ``Nernst``. Arbor implements it as a mechanism and
```

Comment:
```suggestion
Both Arbor and NEURON make use of ``nernst``. Arbor implements it as a mechanism and
```
The name of the scientist and equation is Nernst, correct, but the mechanism implementing
the equation is `nernst`.